api/src/main/java/org/pmiops/workbench/institution/InstitutionEmailDomainMapper.java

```diff
@@ -0,0 +1,33 @@
+package org.pmiops.workbench.institution;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.mapstruct.Mapper;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbInstitutionEmailDomain;
+import org.pmiops.workbench.model.Institution;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(config = MapStructConfig.class)
+public interface InstitutionEmailDomainMapper {
+  default Set<DbInstitutionEmailDomain> modelToDb(
+      final Institution modelInstitution, final DbInstitution dbInstitution) {
+    return Optional.ofNullable(modelInstitution.getEmailDomains()).orElse(Collections.emptyList())
+        .stream()
+        .distinct()
```

Comment:
For mappers that write to a DB object, maybe document whether or not the caller can immediately call `save()` afterwards, or if anything changes in the database as a side effect. Currently we're not doing that anywhere, though.